discord/commands.py

```diff
@@ -0,0 +1,557 @@
+from models import Variable, User, RwrAccount, RwrAccountStat
+from flask_discord_interactions.models.embed import Field
+from flask_discord_interactions import Message, Permission
+from rwrs import app, cache, db, discord_interactions
+from . import constants, utils, embeds
+from rwr.player import Player
+import steam_helpers
+import rwr.scraper
+import arrow
+import os
+
+maintenance_command_group = discord_interactions.command_group(
+    'maintenance',
+    default_permission=False,
+    permissions=[
```

Comment:
Create helper `utils.superadmin_permissions()` which return an array of permissions for me only

Reply:
`utils.admin_permissions()` too, which returns myself + permission for role "181171868517138434" (developers) on Discord